pkg/blob/blob.go

```diff
@@ -415,7 +415,7 @@ func (d *Driver) GetAuthEnv(ctx context.Context, volumeID, protocol string, attr
 		case storageAuthTypeField:
 			azureStorageAuthType = v
 			authEnv = append(authEnv, "AZURE_STORAGE_AUTH_TYPE="+v)
-		case storageIentityClientIDField:
+		case storageIdentityClientIDField:
```

Comment:
can you rebase your PR? typo should be fixed already.

Reply:
done